### DISCUSSION TITLE
Implement language change for wiki

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#22063] [Ideas] [answered]
In-game wiki is nothing for non-english speakers without language change or at least the same language support as set game-wide.

**Comment by frenzibyte** [comment]
Before I start adding this, do we want to follow a similar UX to how web does it (a language dropdown in wiki that translate the page to the selected language)? Or should we force the wiki to the language the user currently has selected in game settings?

**Reply by Loreos7** [reply]
I guess forcing the language selected game-wide would feel more natural and logical

**Reply by peppy** [reply]
Using the game language sounds fine until there's an actual request to do otherwise. Keep it simple.

**Comment by Loreos7** [comment, marked as answer]
https://github.com/ppy/osu/pull/23296
